.github/workflows/focus-ios-ui-tests-previous-os.yml

```diff
@@ -15,12 +15,16 @@ jobs:
     runs-on: macos-14
     strategy:
       fail-fast: false
+      max-parallel: 1
       matrix:
         include:
           - ios_version: 16.4
             ios_simulator: iPhone 14
           - ios_version: 15.5
             ios_simulator: iPhone 13
+          # - ios_version: 18.1
+          #   ios_simulator: iPhone 15
```

Comment:
Yes, 18.1 and 18.0 runtime and simulators are available on macos-14 runner. We may add them later.